src/js/common/schemaform/save-load-actions.js

```diff
@@ -244,9 +246,15 @@ export function fetchInProgressForm(formId, migrations, prefill = false) {
       // If we’ve made it this far, we’ve got valid form
 
       let formData;
+      let metadata;
       try {
         // NOTE: This may change to be migrated in the back end before sent over
-        formData = migrateFormData(resBody.form_data, resBody.metadata.version, migrations);
+        const dataToMigrate = {
+          formId,
+          formData: _.clone(resBody.form_data),
```

Comment:
I don't think we need to clone these. I can't see any errors it would help us avoid.

Reply:
Thanks @jbalboni, change made.